DANS-KNAW/easy-license-creator
# License as a service

No JIRA issue.

#### When applied it will
* Expose the license creator as a RESTful service

#### Where should the reviewer @DANS-KNAW/easy start?
Perhaps [here](https://github.com/DANS-KNAW/easy-license-creator/compare/master...janvanmansum:LICENSE_AS_A_SERVICE#diff-47005b92a437c3545210c8e9a888ffc6R36)

#### How should this be manually tested?
* Install it on `deasy` and check it with `sudo service easy-license-creator status`, etc.
* `curl -X POST http://localhost:8060/create?datasetId=easy-dataset:1` on `deasy`.
